src/plugin.ts

```diff
@@ -182,4 +182,4 @@ const registerDurationMetricsPlugin = (on: Cypress.PluginEvents) => {
   }
 };
 
-export = registerDurationMetricsPlugin;
+export default registerDurationMetricsPlugin;
```

Comment:
I believe this format was causing issues when plugin was loaded in non esm context. What was your exact issue when adding this plugin?

Reply:
@costalonga any updates on this?